uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/LicenseTest.java

```diff
@@ -61,17 +64,21 @@
 	@Parameters
 	public static Collection<String> setupData() {
 		new ShellMenuItem(new WorkbenchShell(), "Help", "About Red Hat JBoss Developer Studio").select();
-		new PushButton("Installation Details").click();
-		new DefaultShell("Red Hat JBoss Developer Studio Installation Details");
-		new DefaultTabItem("Installed Software").activate();
+		Shell shell = new DefaultShell("About Red Hat JBoss Developer Studio");
+		new WaitUntil(new ShellIsAvailable(shell));
```

Comment:
This is not necessary since the previous line did this. Note that
```
new DefaultShell("Title");
```
waits for a shell with the title "Title".